power/Power.cc

```diff
@@ -1204,7 +1204,14 @@ Power::findActivity(const Pin *pin)
     PwrActivity &activity = activity_map_[pin];
     if (activity.origin() != PwrActivityOrigin::unknown)
       return activity;
+  } else if (user_activity_map_.hasKey(pin)) {
+    PwrActivity &activity = user_activity_map_[pin];
+    if (activity.origin() != PwrActivityOrigin::unknown)
+      return activity;
   }
+  const char *pname = network_->pathName(pin);
+  if (strstr(pname, "/IQ") == nullptr)
```

Comment:
Could we instead check whether the pin is a "real" pin or an internal pin? That would be a more general solution than this.

Reply:
At the very least put a TODO

Reply:
Adding a TODO to remove this code altogether, this is internal debugging while in dev mode.